ansible/roles/stack-sunbird/templates/sunbird_nodebb.env

```diff
@@ -1,10 +1,17 @@
 # Initialization variable for mongo
 # Ref: https://community.nodebb.org/topic/4325/set-up-nodebb-via-environment-variables
-database=mongo
+database=redis
+# redis database number
+redis_database={{nodebb_redis_database}}
```

Comment:
This should have some default value. Please reach out to KP team and ask the db numbers they are using. Then we can have a default DB number. 
Example
`redis_database={{nodebb_redis_database | default(1)}}`

Reply:
ok

Reply:
Had a call with KP team member and he told me to use 4